packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="range" type="range" class="sf-range" :disabled="disabled">
+    <slot v-bind="$attrs" />
+  </div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    value: {
```

Comment:
I don't think this is a good name for that prop. Maybe `numberOfSliders`?

Reply:
I think the description for this property is wrong. It should be that this are starting values for sliders and when there is one number given only one slider appears.